frontend/app/abtests/BundleVariant.scala

```diff
@@ -49,8 +48,8 @@ object BundleTier {
 
 
 object Distribution {
```

Comment:
The distribution object was thought to store the existing designs. Before there were A and B, in this PR should exist only A and indicate the different version of A using the `priceindex`.